dso/compliancedb/db.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+import sqlalchemy
+
+import ci.util
+from dso.compliancedb.model import Base
+
+
+class ComplianceDB:
+    def __init__(
+        self,
+        username: str,
+        password: str,
+        hostname: str,
+        port: int,
+        dialect: str = 'postgresql',
+    ):
+        self._engine = sqlalchemy.create_engine(
+            f'{dialect}://{username}:{password}@{hostname}:{port}',
+            echo=True,
+            future=True,
+        )
+        self.Base = Base
+        # we configured our own root logger and use log propagation
+        # therefore pop streamhandler to not have duplicate output
+        logging.getLogger('sqlalchemy.engine.Engine').handlers.pop()
+        self.Base.metadata.create_all(self._engine)
+        self.Session = sqlalchemy.orm.Session(self._engine)
+
+
+class ComplianceDBFactory:
```

Comment:
really..?

rather put it into `ccc.compliancedb`, please. use a plain function, please. this is not enterprise-java..